cloud/cloudstack/cs_volume.py

```diff
@@ -409,7 +410,7 @@ def main():
     argument_spec.update(dict(
         name = dict(required=True),
         disk_offering = dict(default=None),
-        display_volume = dict(choices=BOOLEANS, default=True),
+        display_volume = dict(choices=BOOLEANS, default=None),
```

Comment:
instead of using `choices=BOOLEANS`, set `type=bool`, we'll be updating most modules to make this change.